tests/test_selection/test_base_selection_functions.py

```diff
@@ -238,10 +251,24 @@ def test_find_feature_importance(df_test):
         ],
     )
 
-    mean_, std_ = find_feature_importance(X, y, rf, cv, scoring)
+    mean_, std_ = find_feature_importance(
+        X=X,
+        y=y,
+        estimator=rf,
+        cv=cv,
+        groups=None,
+        scoring=scoring,
+    )
     pd.testing.assert_series_equal(mean_.round(2), expected_mean)
     pd.testing.assert_series_equal(std_.round(4), expected_std)
 
-    mean_, std_ = find_feature_importance(X, y, rf, cv.split(X, y), scoring)
+    mean_, std_ = find_feature_importance(
+        X=X,
+        y=y,
+        estimator=rf,
+        cv=cv.split(X, y),
+        groups=None,
```

Comment:
I don't think we need to add these changes where groups is None. Could we revert back to the original?

What we do need is a test for when groups is not None. Could we add that?

Reply:
yes you are right, as groups=None is default behaviour we do not need to specify it, but rather test when actual groups is provided. I will try to cover tests in the following days

Reply:
as an update to this test, the current format of the `find_feature_importance` and `single_feature_performance` is that none of the arguments have default value, therefore in these function I also didn't assign default None, I can either keep the format, and revert tests back to have groups=None, or assign None inside functions themselves.

Reply:
can you please confirm if this style is ok for `single_feature_importance` and `find_feature_importance` at `base_selection_functions.py`?

```Python
def single_feature_performance(
    X: pd.DataFrame,
    y: pd.Series,
    variables: List[Union[str, int]],
    estimator,
    cv,
    scoring,
    groups=None,
):
````

and

```Python
def find_feature_importance(
    X: pd.DataFrame,
    y: pd.Series,
    estimator,
    cv,
    scoring,
    groups=None,
):
```

If I do not add default None value here, we have to pass groups=None in test files where we do not test groups

Reply:
this is good, thanks!